README.md

```diff
@@ -96,6 +96,21 @@ Run all the tests (unittests only currently) with:
 ```sh
 tox
 ```
+## Proxy Support
+There are 2 ways you can use an HTTP(S) proxy. The first method is using environment variables:
```

Comment:
The interrogation of the env variables happens in the _requests_ framework but I did mention this in the README and also provided a link to the docs.